website/content/docs/commands/volume/claim-delete.mdx

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+page_title: 'Commands: volume claim delete'
```

Comment:
```suggestion
page_title: 'nomad volume claim delete reference'
```

I'm in the process of updating page titles to match the [reference page format](https://hashicorp.atlassian.net/wiki/x/goDsyw) created by the Education team. I don't expect engineers to follow these content guidelines - I'm happy to make the updates for you.